auctioneerrpc/auctioneer.proto

```diff
@@ -564,12 +571,6 @@ message OrderMatchFinalize {
     The final transaction ID of the published batch transaction.
     */
     bytes batch_txid = 2;
-
-    /*
-    The current block height at the time the batch transaction was published to
-    the network.
-    */
-    uint32 height_hint = 3;
```

Comment:
Is this a breaking change? Off top, can't think of anything that relies on this field as is.

Reply:
Yes, it wasn't being used by traders at all. I think rather than removing it though, we need to mark it as reserved in case we decide to extend this proto message in the future?